doc/api/buffer.md

```diff
@@ -2508,6 +2508,9 @@ encoding to another. Returns a new `Buffer` instance.
 Throws if the `fromEnc` or `toEnc` specify invalid character encodings or if
 conversion from `fromEnc` to `toEnc` is not permitted.
 
+Encodings supported by `buffer.transcode()` include: `'ascii'`, `'utf8'`,
```

Comment:
`include` -> `are` assuming the list is supposed to be a complete list of supported encodings.

Reply:
Feel free to making whatever editorial changes you'd like

Reply:
Wanted to confirm the list is supposed to be complete before doing it. I'll take that as a "yes, it's a complete list" and make the change now. 👍